### PR TITLE
NMS-12681: Fix  Reloading the Pollerd daemon causes multiple nodeDown messages

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12681
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

**Comment by cgorantla** [comment]
Looks like this fix doesn't cover all cases. Checking more.

File: opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java [hunk 1]
```diff
@@ -421,14 +421,14 @@ private void scheduleExistingServices() throws Exception {
...
      * @param nodeId a int.
-     * @param nodeLabel a {@link java.lang.String} object.
-     * @param nodeLocation a {@link java.lang.String} object.
-     * @param ipAddr a {@link java.lang.String} object.
-     * @param svcName a {@link java.lang.String} object.
+     * @param nodeLabel a {@link String} object.
+     * @param nodeLocation a {@link String} object.
+     * @param ipAddr a {@link String} object.
+     * @param svcName a {@link String} object.
+     * @param pollableNode
```
Comment (j-white) [inline opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:429]: Explain the purpose of the `pollableNode` argument.

File: opennms-services/src/main/java/org/opennms/netmgt/poller/PollerEventProcessor.java [hunk 2]
```diff
@@ -754,6 +754,7 @@ private void serviceReschedule(Long nodeId, String nodeLabel, String nodeLocatio
             }
         }
 
+        PollStatus pollStatus = pnode != null ? pnode.getStatus() : null;
```
Comment (j-white) [inline opennms-services/src/main/java/org/opennms/netmgt/poller/PollerEventProcessor.java:757]: This value used?

File: opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java [hunk 3]
```diff
@@ -426,7 +426,7 @@ private void scheduleExistingServices() throws Exception {
      * @param nodeLocation a {@link String} object.
      * @param ipAddr a {@link String} object.
      * @param svcName a {@link String} object.
-     * @param pollableNode
+     * @param pollableNode a {@link PollableNode} object
```
Comment (j-white) [inline opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:429]: Not quite what I had in mind :)

Comment (cgorantla) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:429]: Of course not, I have added this comment. I can extend it bit more in detail. https://github.com/OpenNMS/opennms/pull/2988/commits/dc3d858cde36390b5e77ada1a0f40f1031619756#diff-ce272af2d0a2ea89b38de1d671441e79R443

Comment (cgorantla) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:429]: Added more detailed comment.

**Comment by cgorantla** [comment]
> Looks like this fix doesn't cover all cases. Checking more.

Addressed this and now it works for me.

**Comment by cgorantla** [comment]
Thanks @agalue

**Pull request merged**